libraries/WiFi/src/WiFiClient.cpp

```diff
@@ -224,6 +224,11 @@ size_t WiFiClient::write(const uint8_t *buf, size_t size)
     return res;
 }
 
+size_t WiFiClient::write_P(PGM_P buf, size_t size)
+{
+    return write(buf, size);
+}
+
```

Comment:
What would happen if we separate code like it's with WiFi? will all of the source files be included even if not used?